health/src/proto.rs

```diff
@@ -22,7 +22,6 @@ mod protobuf_v3 {
 #[cfg(feature = "protobuf-codec")]
 mod reexports {
     pub use super::protobuf::health::*;
-    pub use super::protobuf::health_grpc::*;
```

Comment:
Again, it causes an "unused import" warning which fails CI. In this case it is because `health::protobuf::health` already re-exports the contents of `health::protobuf::health_grpc`, so the line above already covers re-exporting it here.